### ISSUE TITLE
Issues with Google Drive File Stream & images in Version 0.9.9.27.1 (3305)

### ISSUE THREAD

**Issue by manicsparkle** [typora/typora-issues#2697]
I'm using Version 0.9.9.27.1 (3305) on macOS Mojave 10.14.5. The files I need to work with are stored in a Google Drive folder that is synced locally to my computer with Google Drive File Stream. Before this latest update, I was able to view images in Typora, but now the markdown displays, but it is unable to load the images. I've confirmed that the markdown is correct with other viewers, including the markdown preview in vsCode.

**Comment by abnerlee** [comment]
what does the image link / address look like?

**Comment by manicsparkle** [comment]
The link is using a relative path. I also have Typora configured to store images that are copied or drag and dropped into the document in an images folder under the directory the .md file is in.

So the link looks like this:
```
![robgreen-chat-20190805](./images/robgreen-chat-20190805.png)
```

You can see the pop up that says the image cannot be loaded:
<img width="577" alt="Screen Shot 2019-08-06 at 10 40 17 AM" src="https://user-images.githubusercontent.com/957129/62554474-cd159800-b836-11e9-9c88-0696d2c31082.png">

However, when I view the same .md file in Mark Text, it loads the image just fine:
<img width="485" alt="Screen Shot 2019-08-06 at 10 47 11 AM" src="https://user-images.githubusercontent.com/957129/62554925-a572ff80-b837-11e9-9157-c748ba204702.png">

It also displays correctly in vsCode Markdown Preview.

This seems to be an issue with files stored on Google Drive that are synced locally with the Google Drive File Stream client. When I copy the folder with the markdown files and images to my desktop or other local only location, the images display correctly.

I've also tried specifying the save location with YAML Front-Matter as follows, but that doesn't seem to work either:
```
typora-copy-images-to: ./images
```

Is there a way to gather more detailed logs? I'd be happy to help fix this if there's a way for me to get those details to you.

**Comment by abnerlee** [comment]
You could check if there are image load errors in web console. 

You could open this by 
1. enable "debug mode" in help menu
2. open "Inspect Element" from context menu
3. switch to "console" tab in DevTools

**Comment by abnerlee** [comment]
Also, what is the FULL image path look like?

**Comment by manicsparkle** [comment]
In the case mentioned earlier, the full path to the image is:

```
/Volumes/GoogleDrive/My Drive/uninotes/images/robgreen-chat-20190805.png
```

The .md files are in the "uninotes" directory.

I also tried the following variations, but they didn't work either:
```
![robgreen-chat-20190805](./images/robgreen-chat-20190805.png)

![robgreen-chat-20190805](/Volumes/GoogleDrive/My\ Drive/uninotes/images/robgreen-chat-20190805.png)

![robgreen-chat-20190805](file:///Volumes/GoogleDrive/My%20Drive/uninotes/images/robgreen-chat-20190805.png)
```

I wouldn't expect the space in the path to cause problems. Due to policy in our EDU Google account, I'm unable to change the name of the "My Drive" folder.

This a screenshot of the console with the sections for the image that will not load expanded.
<img width="1045" alt="Screen Shot 2019-08-08 at 8 56 23 AM" src="https://user-images.githubusercontent.com/957129/62710279-a3867900-b9bc-11e9-9b36-705e1d18bc16.png">

I also tried a test in a directory in our departments shared drives section in Google Drive. This is at the same level on my workstation as "My Drive" This had the same results and similar errors in the console.
<img width="432" alt="Screen Shot 2019-08-08 at 9 08 25 AM" src="https://user-images.githubusercontent.com/957129/62710310-ad0fe100-b9bc-11e9-894b-fd7f55fbd908.png">
